src/lib/features/feature-toggle/feature-toggle-service.ts

```diff
@@ -793,6 +793,33 @@ class FeatureToggleService {
         auditUser: IAuditUser,
         user?: IUser,
     ): Promise<void> {
+        if (
+            this.flagResolver.isEnabled(
+                'optionallyDisableWithStrategiesCheckOnly',
```

Comment:
This flag name is difficult to remember and use in conversations. Maybe simplifyOptionalDisableFeature or simplifyDisableFeature?

Reply:
simplifyDisableFeature sounds good, thanks! I wasn't sure what to use, so I picked a hard one to provoke feedback 😁